package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cfpb-chart-builder",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Do we want to do the bumping in the same PR? Okay if we're sure we'll be releasing right after this is merged.

Reply:
Reverted while I debug the failure in npm